### PR TITLE
[model-translation] 📦 🛢 Add model field translations

### PR DESCRIPTION
**Pull request by wlonk**
kitten

### CODE REVIEW

File: metadeploy/adminapi/api.py [hunk 1]
```diff
@@ -137,13 +137,19 @@ class ProductSlugViewSet(AdminAPIViewSet):
 
 
 class PlanStepSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(required=False)
```
Comment (davisagli) [inline metadeploy/adminapi/api.py]: name should be required

File: metadeploy/adminapi/api.py [hunk 2]
```diff
@@ -137,13 +137,19 @@ class ProductSlugViewSet(AdminAPIViewSet):
...
     class Meta:
         model = models.Step
         exclude = ("id", "plan")
 
 
 class PlanSerializer(AdminAPISerializer):
     steps = PlanStepSerializer(many=True, required=False)
+    post_install_message = serializers.CharField(required=False)
+    preflight_message = serializers.CharField(required=False)
+    title = serializers.CharField(required=False)
```
Comment (davisagli) [inline metadeploy/adminapi/api.py]: title should be required. For the others, is required=False okay even though the fields don't have null=True?

Comment (davisagli) [reply metadeploy/adminapi/api.py]: Do we need to be explicit about the serializers for Product too? There isn't full functional test coverage of the admin API yet.

Comment (wlonk) [reply metadeploy/adminapi/api.py]: @davisagli I'm not sure, because I don't know where and how Product serializers might be used. Can you tell me more?

Comment (davisagli) [reply metadeploy/adminapi/api.py]: By the ProductViewSet at /admin/rest/products

Comment (wlonk) [reply metadeploy/adminapi/api.py]: Sorry, I meant that in a slightly different way; I don't know if the intent is to populate those fields or read those fields (`title`, `short_description`, `description`, `click_through_agreement`) via the admin API. I'll assume it is, and just add them to the thing explicitly, but I think we should have more admin API test coverage.

Comment (davisagli) [reply metadeploy/adminapi/api.py]: The intent of the admin API is to provide superusers with read and write access to all the fields. I agree about test coverage and am sorry it's not there yet. In practice we aren't using the API to populate products yet, so it's not high priority.

**Pull request merged**